shell/platform/android/io/flutter/embedding/engine/renderer/FlutterRenderer.java

```diff
@@ -41,9 +41,18 @@
   private final FlutterJNI flutterJNI;
   private final AtomicLong nextTextureId = new AtomicLong(0L);
   private RenderSurface renderSurface;
+  private boolean hasRenderedFirstFrame = false;
```

Comment:
I get that you're saving this via bundles in the fluttersplashview. Why are you saving it again here?

Reply:
I'm not sure what you mean by "saving"? Do you mean caching as a variable? It's returned by `hasRenderedFirstFrame()`...

Reply:
never mind, the SavedState thing is 2 staged. You had to figure out both whether the current view instance rendered the first frame and whether another instance rendered the first frame before.